docs/how-to-upload-manifest.md

```diff
@@ -0,0 +1,120 @@
+# How To Push Operator Manifest To App Registry
+This document shows how one can push operator manifest(s) into `quay.io` or any other compatible app registry server.
+
+```
+The example script(s) in this document refers to `quay.io`
+```
+
+## Format of Operator Manifest
+```
+As proof of concept, we are storing the operator manifest into one file. It could change in the future.
+``` 
+
+The operator manifest file has the following structure -
+```yaml
+name: certified operators
+publisher: redhat
+
+data:
+  customResourceDefinitions: |-
+  clusterServiceVersions: |-
+  packages: |-
+```
+
+## Pre-Requisite
+```
+You need to have an account with quay.io. If you don't have one you can sign up for it. 
+```
+
+### Obtain Access Token
+All API requests made by executing HTTP verb (GET, POST, PUT or DELETE) against the API endpoint URL require an `Authorization header` with an access token. By logging in to `quay.io` you will receive an access token.  
+
+* Save the following excerpt into a file, for example `login.sh`  
+```bash
```

Comment:
Please create a scripts directory and reference this script from there?

Reply:
See my comments below.